### PR TITLE
Devolutions.RemoteDesktopManagerFree version 2021.2.13.0

### PR DESCRIPTION
**Pull request by krystofkrticka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29297)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Comment by krystofkrticka** [comment]
@ItzLevvie what should I change, I only updated manifest which was already I repo.

File: manifests/d/Devolutions/RemoteDesktopManagerFree/2021.2.13.0/Devolutions.RemoteDesktopManagerFree.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- vrb
+- rde
+- psafe3
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://cdn.devolutions.net/download/Setup.RemoteDesktopManagerFree.2021.2.13.0.msi
+  InstallerSha256: 4F537DC136D99775E09262691F81885DCE08D17878A864A864C247C9878D7F16
```
Comment (ItzLevvie) [inline manifests/d/Devolutions/RemoteDesktopManagerFree/2021.2.13.0/Devolutions.RemoteDesktopManagerFree.installer.yaml:31]:
```suggestion
  InstallerSha256: 415CCA95254F770D542F8B3E2274604D229C9B65D5140305DF5321142F7B2FB6
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**